### PR TITLE
docs(Button): Updated documentation template

### PR DESCRIPTION
**Pull request by MarcosViniciusPC**
## Description
https://jirasoftware.catho.com.br/browse/QTM-367

## Review guide
- [ ] Code review
- [ ] Documentation

### CODE REVIEW

File: stories/Button/Button.story.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+### Secondary
+
+<Canvas>
+  <Story story={Secondary}/>
+</Canvas>
+
+### With Icons
+
+The full catalogue of icons can be found [here](?path=/docs/foundation-icons--page#catalog)
```
Comment (toncabral88) [inline stories/Button/Button.story.mdx:66]: Nice, but in `ArgsTable` link is incorrect.

**Review by toncabral88 — CHANGES_REQUESTED** [review]

File: stories/Button/Button.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  disabled: true,
+  chidlren: 'Locked',
+};
+
+export const WithDoneIcon = Template.bind({});
+WithDoneIcon.args = {
+  icon: 'done',
+  chidlren: 'Done',
+  skin: 'warning',
+};
```
[the remark this answers is not in the document]
Comment (MarcosViniciusPC) [reply stories/Button/Button.stories.jsx:77]: OK, I think it's valid

File: stories/Button/Button.story.mdx (hunk 1 above)
Comment (MarcosViniciusPC) [reply stories/Button/Button.story.mdx:66]: in this case the link should be without the #catalog?

File: stories/Button/Button.stories.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (MarcosViniciusPC) [reply stories/Button/Button.stories.jsx:77]: One question, do you think it's interesting to also include stories with sizes?

File: stories/Button/Button.story.mdx (hunk 1 above)
Comment (toncabral88) [reply stories/Button/Button.story.mdx]: No, here is ok, but on the proptypes definition link should be adjusted

File: stories/Button/Button.stories.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (toncabral88) [reply stories/Button/Button.stories.jsx]: Yes

**Review by toncabral88 — APPROVED** [review]
[x] Code review
[x] Documentation updated

**Review by rapahaeru — APPROVED** [review]

**Pull request merged**